programs/castle-vault/src/instructions/init_port_additional_state.rs

```diff
@@ -29,7 +29,8 @@ pub struct InitializePortAdditionalState<'info> {
     pub system_program: Program<'info, System>,
 }
 
-pub fn handler(ctx: Context<InitializePortAdditionalState>, bump: u8) -> ProgramResult {
-    ctx.accounts.vault.vault_port_additional_state_bump = bump;
+pub fn handler(ctx: Context<InitializePortAdditionalState>) -> Result<()> {
+    ctx.accounts.vault.vault_port_additional_state_bump =
+        *ctx.bumps.get("port_additional_states").unwrap();
```

Comment:
Can you remove the unwrap here?

Reply:
anchor tests make me think this is how youre supposed to get bumps now
https://github.com/coral-xyz/anchor/blob/master/tests/chat/programs/chat/src/lib.rs#L15